app/experimenter/experiments/models.py

```diff
@@ -92,6 +93,8 @@ class Experiment(ExperimentConstants, models.Model):
         validators=[MaxValueValidator(ExperimentConstants.MAX_DURATION)],
     )
 
+    is_multi_pref_format = models.BooleanField(default=False)
```

Comment:
I think in the name of consistency this could just be `is_multi_pref` like `is_branched_addon`?

Reply:
ooohh so, this should be `is_multi_pref` and `is_multi_pref_experiment` should be `use_multi_pref_serializer`?